src/lib/components/header/header.svelte

```
@@ -13,6 +13,7 @@
 	import HeaderLink from "./header-link.svelte";
 	import { useSession, useSignIn, useSignOut } from "$lib/session";
```

ESLint

[error] 'useSession' is defined but never used.

(@typescript-eslint/no-unused-vars)